### DISCUSSION TITLE
Import/Export feature for the Chat Color plugin

### DISCUSSION THREAD

**Discussion by nmehta94** [runelite/runelite#16071] [Ideas]
This would be extremely helpful for sharing chat color settings with users/allow users to back up their settings in case. Manually entering each hex value and then saving a screenshot of them somewhere for that purpose has been pretty annoying, and wanted to manually copy/paste config values to get around this but was advised in the RL Discord to not do this. This feature would be super nice imo!

**Comment by raiyni** [comment]
You can already backup your settings with the info plugin.

**Reply by nmehta94** [reply]
which info plugin? does that help with exporting your settings to share as is possible with ground markers?
